src/components/styled/drawer.css

```diff
@@ -1,6 +1,7 @@
 .drawer {
   width: 100%;
   &-side {
+    @apply overflow-hidden;
```

Comment:
overflow hidden is not the solution. if sidebar content is long, it should overflow.

Reply:
This is the problem .
<img width="946" alt="image" src="https://github.com/saadeghi/daisyui/assets/4584599/2c3d650a-3cee-42b8-aec2-d21b639aeb75">

>overflow hidden is not the solution. if sidebar content is long, it should overflow. 

yeah I see, so my solution is `@apply overflow-x-hidden;`  @saadeghi